tests/test_dataset.py

```diff
@@ -141,18 +144,10 @@ def testExistingOutputGen3(self):
         outputDir = os.path.join(testDir, 'badOut')
 
         try:
-            os.makedirs(outputDir)
-            output = os.path.join(outputDir, 'foo.txt')
-            with open(output, 'w') as dummy:
-                dummy.write('This is a test!')
-
             self._testbed.makeCompatibleRepoGen3(outputDir)
-            self.assertTrue(os.path.exists(outputDir), 'Output directory must exist.')
-            self.assertTrue(os.listdir(outputDir), 'Output directory must not be empty.')
-            self.assertTrue(os.path.exists(os.path.join(outputDir, 'butler.yaml')),
-                            'Output directory must have a butler.yaml file.')
-            butler = dafButler.Butler(outputDir)
-            self.assertIn("LSST-ImSim/calib", butler.registry.queryCollections())
+            self._checkOutputGen3(outputDir)
+            self._testbed.makeCompatibleRepoGen3(outputDir)
+            self._checkOutputGen3(outputDir)
```

Comment:
This second check is making sure that the previous line didn't delete it?

Reply:
Making sure nothing's been invalidated. I don't see any point in filtering out tests on the grounds that "surely that couldn't break".